classes/python3.yaml

```diff
@@ -0,0 +1,48 @@
+buildTools: [python3]
+buildVars: [BOB_HOST_PLATFORM]
+buildScript: |
+    if [[ "$BOB_HOST_PLATFORM" == "linux" ]] ; then
+        cat << EOF > getdestlib.py
+    #!/usr/bin/env python3
+    import sysconfig
+    print(sysconfig.get_config_var('DESTLIB'))
+    EOF
+        DESTLIB="$(python3 getdestlib.py)"
+    else
+        DESTLIB="/usr/lib"
+    fi
+
+    for i in "${!BOB_DEP_PATHS[@]}" ; do
+        if [[ $i == *-dev ]] ; then
+            PREFIX="${BOB_DEP_PATHS[$i]}$DESTLIB"
+            for j in "" "lib-dynload" "site-packages" ; do
+                PYTHONPATH="${PYTHONPATH:+$PYTHONPATH:}$PREFIX${j:+/$j}"
+            done
+        fi
+    done
+    PREFIX="${BOB_TOOL_PATHS[python3]}$DESTLIB"
+    for j in "" "lib-dynload" "site-packages" ; do
+        PYTHONPATH="${PYTHONPATH:+$PYTHONPATH:}$PREFIX${j:+/$j}"
+    done
+    export PYTHONPATH
+
+    pip_install()
+    {
+        # force lib instead of Lib for MSYS
+        mkdir -p install/usr/lib
+        python3 -m pip install $* \
+            --root ${PWD}/install \
+            --prefix usr \
```

Comment:
do not use "/usr". this will fail in MSYS, but just "usr" will also work in linux!